src/main/python/server.py

```diff
@@ -16,15 +18,43 @@
 
 
 class Service(service_pb2_grpc.FeatureExtractorServicer):
-    pass
+    """Feature Extractor Service"""
+
+    def Identifiers(self, request, context):
+        """Extract identifiers weighted set"""
+
+        extractor = IdentifiersBagExtractor(
+            docfreq_threshold=request.docfreqThreshold, split_stem=request.splitStem)
```

Comment:
From what I can see, all `BagsExtractor`s, of which `IdentifiersBagExtractor` is a [particular kind](https://github.com/src-d/ml/blob/master/sourced/ml/extractors/identifiers.py#L7), have a [`weight` parameter](https://github.com/src-d/ml/blob/master/sourced/ml/extractors/bags_extractor.py#L18) - do you think it should be added on our side as well?

AFAIK it's something that most probably going to be used as part of parameter adjustment as soon as the values are inferred from a labeled data, provided by 🐈 .

Reply:
oh. I'm not sure how I missed it in the task about creating proto files. My bad.

Do you mind if I add this parameter in a separate PR? It should be in all extractors and requires changes in proto & scala.